blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React, { Component, PropTypes } from 'react';
+import { Icon } from 'react-material-icons-blue';
+import { ReadableDate } from '@jenkins-cd/design-language';
+import { LiveStatusIndicator } from '@jenkins-cd/design-language';
+import { TimeDuration } from '@jenkins-cd/design-language';
+import moment from 'moment';
+
+const { object, func } = PropTypes;
+
+class RunDetailsHeader extends Component {
+    handleAuthorsClick() {
+        if (this.props.onAuthorsClick) {
+            this.props.onAuthorsClick();
+        }
+    }
+
+    handleOrganizationClick() {
+        if (this.props.onOrganizationClick) {
+            this.props.onOrganizationClick();
+        }
+    }
+
+    handleNameClick() {
+        if (this.props.onNameClick) {
+            this.props.onNameClick();
+        }
+    }
+
+    render() {
+        const {
+            data: {
+                id,
+                name,
+                organization,
+                pipeline,
+                changeSet,
+                result,
+                state,
+                durationInMillis,
+                endTime,
+                startTime,
+                estimatedDurationInMillis,
+                commitId,
+
+            },
+        } = this.props;
```

Comment:
Isn't the above the same as ...

```javascript
const {
    id,
    name,
    organization,
    pipeline,
    changeSet,
    result,
    state,
    durationInMillis,
    endTime,
    startTime,
    estimatedDurationInMillis,
    commitId,
} = this.props.data;
```

I dunno, but the above just seems a little clearer to me. Matter of taste I suppose.